lib/Swagger/src/TemplateClassification/Storage/Api/TCSApi.php

```diff
@@ -165,71 +169,65 @@ public function getTemplateDetailsWithHttpInfo($wiki_id, $page_id)
         $resourcePath = str_replace("{format}", "json", $resourcePath);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "{format}" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)